src/diffusers/pipelines/t2i_adapter/pipeline_stable_diffusion_adapter.py

```diff
@@ -183,9 +182,6 @@ def __init__(
                 " checker. If you do not want to use the safety checker, you can pass `'safety_checker=None'` instead."
             )
 
-        if isinstance(adapter, (list, tuple)):
```

Comment:
Don't we need to keep this?

Reply:
MultiAdapter doesn't actually take a second argument in its constructor. If we ever had hit this code path, it would have thrown an error